app/src/main/java/cz/applifting/humansis/model/Country.kt

```diff
@@ -0,0 +1,34 @@
+package cz.applifting.humansis.model
+
+import android.content.Context
+import cz.applifting.humansis.R
+
+data class Country(
+    val iso3: String = "",
+    var context: Context? = null
```

Comment:
Tohle určitě ne. Můžeš tam dát překlad, do kontruktoru a ve třídě country například vytvořit companion, kde bude metoda create a ta přijme v parametru context a použije ho pro vytvoření překladu, ale takhle ne